nimdoc/testproject/subdir/subdir_b/utils.nim

```diff
@@ -50,9 +50,9 @@ template bEnum*(): untyped =
     discard
 
 template fromUtilsGen*(): untyped =
-  ## this should be shown in utils.html
+  ## should be shown in utils.html only
   runnableExamples:
-    assert 3*2 == 6
+    discard "should be in utils.html only, not in module that calls fromUtilsGen"
```

Comment:
Why was this changed? Does the `assert` inside a runnableExample inside a template still work?

Reply:
yes, of course assert still works, the new comment makes it clear when you look at generated file where it came from and where it should end up

Reply:
this is also tested via tests/nimdoc/trunnableexamples.nim btw